backend/models.py

```diff
@@ -48,3 +48,28 @@ class Post(models.Model):
     # FRIENDS should've already been sent the post so they don't need this
     unlisted = models.BooleanField()
     # unlisted means it is public if you know the post name -- use this for images, it's so images don't show up in timelines
+
+
+class Inbox(models.Model):
+    '''
+    Inbox model:
+        id          PRIMARY KEY
+        author      FOREIGN KEY Reference to Author
+        post
+        follow
+        like
+        comments
+        published 
+    
+    '''
+    id = models.UUIDField(unique=True, primary_key=True, default=uuid.uuid4, editabable = False, max_length=100)
+    # Suppose to be Author? Instead of User
+    author = models.ForeignKey(Author, on_delete=models.CASCADE)
+    
+    # I'm retrieving these items? Could group items into a JSONField
+    post = models.ForeignKey(Post, on_delete=models.CASCADE, related_name='post')
```

Comment:
lets use the _id convention for relationships of has one / belongs to one like this. so post_id instead of post